conf/solr/7.3.1/readme.me

```diff
@@ -0,0 +1 @@
+Please see the dev guide for what to do with Solr config files.
```

Comment:
This should probably end in ".md" instead of ".me".

Reply:
I agree - but this was how I found it in the 7.3.0 configs.

Reply:
Weird. Can you please use "md" for 7.3.1?